server/models.py

```diff
@@ -248,7 +263,13 @@ class Inbox(db.Model):
     like = db.Column(db.ForeignKey("like.id"))
     follow = db.Column(db.ForeignKey("requests.id"))
 
-    def __init__(self, owner, post=None, like=None, follow=None):
+    def __init__(
+        self,
+        owner: int,
+        post: int | None = None,

```

Comment:
I think that | is a bitwise operator and not a logical one? Did you mean to use it as a bitwise operator? or did you mean `or`?

Reply:
For type annotations, `|` does mean or HOWEVER I just learned that this is a feature from python 3.10. Since our application supports python >= 3.8, I'll change it to use the Union type.

Reply:
For type annotations, | does mean or HOWEVER I just learned that this is a feature from python 3.10. Since our application supports python >= 3.8, I'll change it to use the Union type.